app/modules/update-tools.js

```diff
@@ -10,7 +10,7 @@ var updateTools = {
   updatesForDisplay: function(quantity) {
     var self = this;
 
-    if(quantity > quantities.updates) {
+    if(!quantity || quantity > quantities.updates) {
```

Comment:
Sure. I have probably developed some stylistic idiosyncrasies of late. Ta for picking me up on it.